packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -239,7 +239,8 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
     const handleMapOptionsChange = (newMapOptions) => {
         setMapOptions(previousMapOptions => ({
             ...previousMapOptions,
-            ...newMapOptions
+            ...newMapOptions,
+            minZoom: 10
```

Comment:
The value 10 for minZoom is used twice. Can we extract it into some constant (file)?